djimporter/tasks.py

```diff
@@ -25,15 +25,18 @@ def run_importer(csv_model, csv_filepath, log_id, context={}):
 
     # run importer
     importer = importer_class(csv_filepath, context=context)
-    importer.is_valid()
+    importer.is_valid(log)
     importer.save()
 
     # update log with import result
     if importer.errors:
         log.status = ImportLog.FAILED
+        log.progress = 0
+        log.num_rows = 0
```

Comment:
I wonder if it will be more useful to keep progress and num_rows before importer fails? (Instead of resetting counter to zero). Or maybe better, log.progress = 100  (task has finished, altough it has errors).

What do you think?

Reply:
I have removed these two lines. It's true that you have finished the task (so log.progrress should be 100), but may be confusing because you can think that the importation has finished ok, although status is not... But if you prefer to set it to 100, no problem.